nats/src/main.rs

```diff
@@ -216,7 +237,12 @@ impl ProviderHandler for NatsMessagingProvider {
     /// If the link is allowed, return true, otherwise return false to deny the link.
     #[instrument(level = "debug", skip(self, ld), fields(actor_id = %ld.actor_id))]
     async fn put_link(&self, ld: &LinkDefinition) -> RpcResult<bool> {
-        let config = ConnectionConfig::new_from(&ld.values)?;
+        // If the link definition values are empty, use the default connection configuration
+        let config = if ld.values.is_empty() {
+            self.default_config.clone()
+        } else {
+            ConnectionConfig::new_from(&ld.values)?
```

Comment:
I think you'll still want to log if this is an error, since the host ignores responses on this topic, right?

Reply:
correct as usual